values/demo.yaml

```diff
@@ -157,3 +157,11 @@ abha-verification:
   metadata:
     labels:
       environment: demo
+
+bahmni-mart:
+  enabled: false
+  config:
+    CRON_TIME: "*/10 * * * *"
```

Comment:
As discussed, keep cron time to 15 or 30 mins

Reply:
we have kept cron time to 5 minutes as of now as discussed.